Documentation/Signoffs/Image_Processing.md

```diff
@@ -0,0 +1,113 @@
+# Image Processing Subsystem
+
+**Function:**
+
+The goal of this subsystem is to receive and process the data from the camera sensor to determine the position and speed of the practice golf ball.  
+
+**Constraints:**
+
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1| Must be able to distinguish the golf ball from surroundings based on golf ball's color| System Requirment|
+| 2| Must be able to extract the x,y coordinates of the golf ball with an inch of accuracy to distinguish between the wires and variable height| System Requirment|
```

Comment:
Your justification says that this is an assumption. That's not sufficient. How do you know the accuracy is appropriate?